### ISSUE TITLE
rtp_receiver_threads parameter problem

### ISSUE THREAD

**Issue by pumper** [sems-server/sems#210]
Hi I compiled sems with instructions of wiki on debian 11 and installed it successfully. it runs and works normally .
I have a virtual machine with 32 core cpu and 32GB of ram and try to use sems for more than 5000 concurrent calls with 1500 cps so i tried to set rtp_receiver_threads=8 and restart sems. it seems everything works but i get alot of errors in sems log like thease:

`
 Dec 21 13:02:25 sbc-dev sems[1060]: [#7f27b9b1b700] [addStream, AmRtpReceiver.cpp:151] ERROR:  trying to insert existing stream [0x7f29a00070f0] with sd=231
 Dec 21 13:02:25 sbc-dev sems[1060]: [#7f27b9b1b700] [addStream, AmRtpReceiver.cpp:151] ERROR:  trying to insert existing stream [0x7f29cc000bf0] with sd=233
 Dec 21 13:02:25 sbc-dev sems[1060]: [#7f27b9b1b700] [addStream, AmRtpReceiver.cpp:151] ERROR:  trying to insert existing stream [0x7f29a00070f0] with sd=231
 Dec 21 13:02:40 sbc-dev sems[1060]: [#7f27b44c5700] [addStream, AmRtpReceiver.cpp:151] ERROR:  trying to insert existing stream [0x7f2ba0003010] with sd=1459
 Dec 21 13:03:29 sbc-dev sems[1060]: [#7f27a2daf700] [addStream, AmRtpReceiver.cpp:151] ERROR:  trying to insert existing stream [0x7f2828037fd0] with sd=1573
 Dec 21 13:05:21 sbc-dev sems[1060]: [#7f2a3fafa700] [addStream, AmRtpReceiver.cpp:151] ERROR:  trying to insert existing stream [0x7f2b08145d10] with sd=1033
 Dec 21 13:08:50 sbc-dev sems[1060]: [#7f27bdf5f700] [addStream, AmRtpReceiver.cpp:151] ERROR:  trying to insert existing stream [0x7f2968011450] with sd=237
 Dec 21 13:09:33 sbc-dev sems[1060]: [#7f27c21a1700] [addStream, AmRtpReceiver.cpp:151] ERROR:  trying to insert existing stream [0x7f2a2c099d90] with sd=247
 Dec 21 13:09:53 sbc-dev sems[1060]: [#7f27ab435700] [addStream, AmRtpReceiver.cpp:151] ERROR:  trying to insert existing stream [0x7f2aa8066e30] with sd=1145
 Dec 21 13:10:19 sbc-dev sems[1060]: [#7f27a61e3700] [addStream, AmRtpReceiver.cpp:151] ERROR:  trying to insert existing stream 
 Dec 21 13:12:37 sbc-dev sems[1060]: [#7f283c2c2700] [addStream, AmRtpReceiver.cpp:151] ERROR:  trying to insert existing stream [0x7f29fc104d10] with sd=949
 Dec 21 13:12:40 sbc-dev sems[1060]: [#7f27a61e3700] [addStream, AmRtpReceiver.cpp:151] ERROR:  trying to insert existing stream [0x7f2aa80008d0] with sd=231
 Dec 21 13:14:16 sbc-dev sems[1060]: [#7f27a60e2700] [addStream, AmRtpReceiver.cpp:151] ERROR:  trying to insert existing stream [0x7f2aa80008d0] with sd=231
 Dec 21 13:14:32 sbc-dev sems[1060]: [#7f27b74f5700] [addStream, AmRtpReceiver.cpp:151] ERROR:  trying to insert existing stream [0x7f28600b3fb0] with sd=1485
`
please help me with this strange problem

**Comment by denyspozniak** [comment]
@pumper Yes, I also encountered this error and as I noticed it correlated with sems crashes.
I ended up eliminating the RTP handling (left only SIP).
If you've determined the cause, share please the information.
